lib/util/stream_utils.js

```diff
@@ -426,6 +429,13 @@ shaka.util.StreamUtils = class {
 
       const audio = variant.audio;
 
+      if (isFirefoxAndroid && audio && audio.encrypted &&
+          audio.codecs.toLowerCase().includes('opus')) {
+        shaka.log.debug('Dropping variant - not compatible with platform',
+            StreamUtils.getVariantSummaryString_(variant));
+        return false;
+      }
```

Comment:
Could be moved to `if(audio)` closure below (along with creating `isFirefoxAndroid`)

Reply:
It is to do it in the same way as it is done on Xbox a few lines above (382).

Reply:
Makes sense!